new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java

```diff
@@ -84,5 +120,64 @@ public void beforeAll(ExtensionContext context) {
   @Override
   public void close() {
     logger.info("Cleanup images after all test suites are run");
+
```

Comment:
@vanajamukkara Did you test this ? when does this method gets invoked, at the end of the first class that creates the image or at the end of the last class that executes ?

Reply:
yes, its getting executed at the very end of the last class. https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests-10/50/consoleFull

Reply:
Great